### ISSUE TITLE
Block style button in form not rendering in Android

### ISSUE THREAD

**Issue by AbigailMcP** [GeekyAnts/NativeBase#1852]
## react-native, react and native-base version
- react 16.3.2
- react-native 0.55.3
- native-base 2.4.2

## Expected behaviour
A block button should be rendered in the Android version of the app.

## Actual behaviour
The button does not render.

## Steps to reproduce (code snippet or screenshot)

Code snippet for the form:

```
const SignInForm = function({ email, password, onChange, onSubmit }) {
  return (
    <Form>
      <Item>
        <Input value={email} onChangeText={onChange.bind(null, 'email')}/>
      </Item>
      <Item>
        <Input value={password} onChangeText={onChange.bind(null, 'password')}/>
      </Item>
      <Button block rounded onPress={onSubmit}>
        <Text>Sign in</Text>
      </Button>
    </Form>
  );
}
```

Used in a screen:

```
const SignInScreen = ({ signup }) => (
  <KeyboardAvoidingView>
    <Container>
      <View padder style={{flex: 1, justifyContent: 'center'}}>
        <SigninForm/>
        <Button transparent onPress={signup}>
          <Text>I don't have a Stately account</Text>
        </Button>
        <Button transparent onPress={() => Linking.openURL(url)}>
          <Text>Forgot password?</Text>
        </Button>
      </View>
    </Container>
  </KeyboardAvoidingView>
);
```
## Screenshot of emulator/device
![screenshot_20180430-100823](https://user-images.githubusercontent.com/18436946/39421141-90786e02-4c5f-11e8-8237-219d4c30059c.png)

## Is the bug present in both ios and android or in any one of them?
Android only.

## Any other additional info which would help us debug the issue quicker.
The transparent block buttons in the SignInScreen component work fine as you can see in the screenshot ('I don't have an account' and 'Forgot password?', but the block button in the SignInForm does not render.

If I remove the 'block' property from the button in the form, then the button will render.

**Comment by kodayashi** [comment]
Hi @SupriyaKalghatgi - I looked at your change and applied it to my local 2.4.5 install, looks good thanks!

**Comment by SupriyaKalghatgi** [comment]
2.5.0

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.5.0](https://github.com/GeekyAnts/NativeBase/tree/v2.5.0)
